### PR TITLE
Add native image configurations

### PR DESCRIPTION
**Pull request by MadhukaHarith92**
## Purpose

Fixes: https://github.com/ballerina-platform/ballerina-standard-library/issues/3481

## Checklist
- [x] Linked to an issue
- [ ] Updated the specification
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: ballerina/Dependencies.toml [hunk 1]
```diff
@@ -1,117 +0,0 @@
-# AUTO-GENERATED FILE. DO NOT MODIFY.
```
Comment (niveathika) [inline ballerina/Dependencies.toml]: Why is dependencies.toml file removed?

File: native/src/main/resources/META-INF/native-image/io.ballerina.stdlib/persist-native/reflect-config.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+[
+{
+  "name":"com.mysql.cj.conf.url.SingleConnectionUrl",
```
Comment (niveathika) [inline native/src/main/resources/META-INF/native-image/io.ballerina.stdlib/persist-native/reflect-config.json:3]: Could u check with @TharmiganK on how to handle configs used for test cases?

File: ballerina/Dependencies.toml (hunk 1 above)
Comment (MadhukaHarith92) [reply ballerina/Dependencies.toml]: Added.

**Comment by MadhukaHarith92** [comment]
Checked on a sample project with a persist entity. The above configurations are not being generated. Need to check how to handle configs used for test cases.

File: native/src/main/resources/META-INF/native-image/io.ballerina.stdlib/persist-native/reflect-config.json (hunk 2 above)
Comment (TharmiganK) [reply native/src/main/resources/META-INF/native-image/io.ballerina.stdlib/persist-native/reflect-config.json:3]: Lets use `ballerinax.mysql-driver` module for testing. And if we add these configs to mysql driver module then we don't need to add the configs here

**Comment by MadhukaHarith92** [comment]
Closing this PR since as per the discussion, the MySQL driver with the native image configs have not been released yet. Need to check once it's released and the test cases are updated.

**Pull request closed without merging**